docs/ignore.md

```diff
@@ -9,6 +9,14 @@ Prettier offers an escape hatch to ignore a block of code or prevent entire file
 
 To exclude files from formatting, add entries to a `.prettierignore` file in the project root or set the [`--ignore-path` CLI option](cli.md#ignore-path).
 
+For example: 
+
+```sh
+# .prettierignore
+# ignores files with an .html extension
+*.html 
```

Comment:
Prettier doesn't format HTML out of the box, so this is a bit misleading- could we maybe change the example to `*.test.js`?

Reply:
(Not that choosing not to format your tests is very common. Not sure what a better glob would be...)

Reply:
`*.min.js`?

Reply:
And/or
```js
dist
package.json
```